src/sfeQwiicUltrasonic.h

```diff
@@ -44,15 +52,15 @@ class sfeQwiicUltrasonic
     /// @return 0 for succuss, negative for errors, positive for warnings
     sfeTkError_t getDistance(uint16_t &distance);
 
-    /// @brief Triggers a new measurement and reads the previous one
-    /// @param distance Distance in mm
+    /// @brief Changes the I2C address of older Qwiic Ultrasonic sensors.
+    /// @param address New address, must be in the range 0x20 to 0x2F
     /// @return 0 for succuss, negative for errors, positive for warnings
-    sfeTkError_t getTriggeredDistance(uint16_t &distance);
+    sfeTkError_t changeAddress(const uint8_t &address);
 
-    /// @brief Changes the I2C address of the Qwiic Ultrasonic sensor
+    /// @brief Changes the I2C address of the latest revision of the Qwiic Ultrasonic sensor.
     /// @param address New address, must be in the range 0x20 to 0x2F
     /// @return 0 for succuss, negative for errors, positive for warnings
-    sfeTkError_t changeAddress(uint8_t &address);
+    sfeTkError_t updateAddress(uint8_t &address);
```

Comment:
The "old" does not have a version in copper and so I just went with "old" and "new" but you're right. I'll refer to the newer one as version v10 since it's in the copper, and if the user does not see a version in copper then they're running the "older" version of the board.